ibu-imager/postpivot/postpivot.go

```diff
@@ -51,6 +52,11 @@ func NewPostPivot(scheme *runtime.Scheme, log *logrus.Logger, ops ops.Ops,
 	}
 }
 
+const (
+	nodeIpFile       = "/run/nodeip-configuration/primary-ip"
+	dnsmasqOverrides = "/etc/default/sno_dnsmasq_configuration_overrides"
+)
+
 func (p *PostPivot) PostPivotConfiguration(ctx context.Context) error {
```

Comment:
This gets executed before or after prepare-installation-configuration.sh?

Reply:
Yep, it is installation-configuration.service and yes my idea to remove prepare-installation-configuration.sh at all at the end